app/controllers/reports_controller.rb

```diff
@@ -142,7 +142,7 @@ def pick_entity()
       if @entity == 'Patient'
         @list = Patient.where(patient_id: params[:query])
         if @list.empty?
-          @list = Patient.where("name LIKE :prefix", prefix: "%#{params[:query]}%")
+          @list = Patient.where("first_name LIKE :prefix or last_name LIKE :prefix", prefix: "%#{params[:query]}%")
```

Comment:
nit: capitalize the "or" please?

I kinda like having the SQL commands/operators be in all caps as a convention.